src/parsers/manifest/dash/common/parse_representations.ts

```diff
@@ -213,6 +213,17 @@ export default function parseRepresentations(
       codecs = codecs === "mp4a.40.02" ? "mp4a.40.2" : codecs;
       parsedRepresentation.codecs = codecs;
     }
+
+    let supplementalCodecs: string | undefined;
+    if (representation.attributes.supplementalCodecs != null) {
```

Comment:
Why don't you use a !==, in order to verify also the type?

Reply:
Yes we now try to avoid relying on double equals in the RxPlayer as:
  - double equals' behavior might not be understood by every dev
  - it's more explicit when reading to have the exact conditions written
  - it avoid some classes of bug where we wrote double equals just by lazyness.
    Here the intent is to have to check explicitely for what can happen. In some situations where only `null` or `undefined` are possible at first, we may create a bug in future code if ever the other value become possible.

There are some areas of the code still relying on the `== null` trick, but that's old logic we ultimately want to replace them by more explicit code.

Reply:
Ok, should I change all the  `!=` to `!isNullOrUndefined` in the `parse_representation file`, or should I do another MR for this change ?

Reply:
As you want, both are good to me

Reply:
Here isn't it only `undefined` though? I'm unsure

Reply:
Yes I think too it's only `undefined` but as a precaution I didn't want to change the behavior of the API.